pubspec.yaml

```diff
@@ -8,10 +8,10 @@ author: Devon Carew <devoncarew@google.com>
 environment:
   sdk: '>=1.10.0 <2.0.0'
 dependencies:
-  args: '^0.13.1'
-  logging: '^0.11.0'
+  args: '^0.13.2'
+  logging: '^0.11.1'
```

Comment:
Changed, but below is my thinking for how I have been maintaining version constraints:

I always update the version constraints to the exact version of the most recent stable version of whatever package I am using. Any ^... style version constraint that is compatible with the something that leaves .0 at the end will work with the more exact version constraint. However by using the tighter constraint I ensure that users will be using the same or newer version of the package as I tested with. Additionally, a tighter constraint should actually improve the performance of pubs constraint solver (fewer options to consider).